fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/CallableCache.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Netflix, Inc.
```

Comment:
s/Netflix/Atlassian?

Reply:
Fixed! Thank you for pointing that out! 👀